okhttp/build.gradle.kts

```diff
@@ -25,6 +25,8 @@ kotlin {
     }
     val commonTest by getting {
```

Comment:
I get the following error trying to move a single file to commonMain. What should commonMain be?

```
Execution failed for task ':okhttp:copyKotlinTemplates'.
> Could not copy file '/Users/yschimke/IdeaProjects/okhttp/okhttp/src/commonMain/kotlinTemplates/okhttp3/Headers.kt' to '/Users/yschimke/IdeaProjects/okhttp/okhttp/build/generated/sources/kotlinTemplates/okhttp3/Headers.kt'.
   > Missing property (line) for Groovy template expansion. Defined keys [projectVersion].
```

Reply:
Ignore, my stupidity.